src/pages/Postcreatepage/index.tsx

```diff
@@ -0,0 +1,98 @@
+                        description: Yup.string().required('Required'),
+                        place: Yup.string().required('Required')
+                    })}
+                    onSubmit={(values) =>{submitHandler(values)}}
+                    >
+                        {({handleSubmit, handleChange, values}) =>(
+                            <Styled.FormWrapper>
+                            <Form onSubmit={handleSubmit}>
+                                <Form.Group>
+                                    <Form.Label>Title</Form.Label>
+                                    <Form.Control
+                                    type='text'
+                                    placeholder="Enter title"
+                                    onChange={handleChange}
+                                    value={values.title}
+                                    name='title' />
+                                </Form.Group>
+                                <ErrorMessage name='title'>{ msg => <div style={{ color: 'red' }}>*{msg}</div> }</ErrorMessage>
+
+                                <Form.Group>
+                                    <Form.Label>Place</Form.Label>
+                                    <Form.Control
+                                    type='text'
+                                    placeholder="Enter place"
+                                    onChange={handleChange}
+                                    value={values.place}
+                                    name='place' />
+                                </Form.Group>
+                                <ErrorMessage name='place'>{ msg => <div style={{ color: 'red' }}>*{msg}</div> }</ErrorMessage>
+
+                                <Form.Group>
+                                    <Form.Label>Description</Form.Label>
+                                    <Form.Control
+                                    as='textarea'
+                                    rows={3}
+                                    placeholder="Enter description"
+                                    onChange={handleChange}
+                                    value={values.description}
+                                    name='description' />
+                                </Form.Group>
+                                <ErrorMessage name='description'>{ msg => <div style={{ color: 'red' }}>*{msg}</div> }</ErrorMessage>
+
+                                <Styled.BtnWrapper>
+                                    <CustomButton buttonVariant='primary' type='submit'>
+                                        Share
+                                    </CustomButton>
+                                </Styled.BtnWrapper>
+
+                            </Form>
+                            </Styled.FormWrapper>
+                        )}
+                </Formik>
+            </Styled.InnerContainer>
+        </CommonLayout>
+    );
+};
+
+const mapDispatchToProps = ( dispatch: any ) => {
```

Comment:
I am also confused about which type to use with this dispatch ?
When I used my created Dispatch type it was giving error.

Reply:
```
import { Dispatch } from 'redux';
const mapDispatchToProps = ( dispatch: Dispatch ) => {
```

you can use it like this here.